src/content/docs/en/reference/error-reference.mdx

```diff
@@ -127,7 +127,6 @@ The following reference is a complete list of the errors you may encounter while
 ## Action Errors
 
 - [**ActionsWithoutServerOutputError**](/en/reference/errors/actions-without-server-output-error/)<br/>Actions must be used with server output.
-- [**ActionsUsedWithForGetError**](/en/reference/errors/actions-used-with-for-get-error/)<br/>An invalid Action query string was passed by a form.
```

Comment:
Should the automation delete the file for this error?

Reply:
Thank you for looking carefully @Fryuni! We intentionally keep the old files around, so that someone running an old version of Astro doesn’t hit a 404 if they encounter this error.